tests/driver_soft_spi/Makefile

```diff
@@ -0,0 +1,17 @@
+APPLICATION = driver_soft_spi
+include ../Makefile.tests_common
+
+BOARD ?= stm32f4discovery
+
+BOARD_WHITELIST := stm32f4discovery samr21-xpro
```

Comment:
Why only these two boards? Shouldn't the `soft_spi` run for every board we support?

Reply:
In theory yes, but it hasn't been tested on many devices yet.